hw02_easy.py

```diff
@@ -0,0 +1,60 @@
+import random
+
+# Задача-1:
+# Дан список фруктов.
+# Напишите программу, выводящую фрукты в виде нумерованного списка,
+# выровненного по правой стороне.
+
+# Пример:
+# Дано: ["яблоко", "банан", "киви", "арбуз"]
+# Вывод:
+# 1. яблоко
+# 2.  банан
+# 3.   киви
+# 4.  арбуз
+
+# Подсказка: воспользоваться методом .format()
+
+fruit_list = ["яблоко", "банан", "абрикос", "персик"]
+for fruit in fruit_list:
+    print('{} {:>10}'.format(fruit_list.index(fruit) + 1, fruit))
+
+
+
+
+# Задача-2:
+# Даны два произвольные списка.
+# Удалите из первого списка элементы, присутствующие во втором списке.
+
+list1 = [1, 2, 4, 4, 5]
+list2 = [2, 4, 6, 8, 10]
+list_del = []
+
+print(list1, list2)
+
+for li in list1:
+    if li not in list2:
+        list_del.append(li)
+list1 = list_del
+
+print("После удаления дубликатов - ", list1, list2)
+
+
+# Задача-3:
+# Дан произвольный список из целых чисел.
+# Получите НОВЫЙ список из элементов исходного, выполнив следующие условия:
+# если элемент кратен двум, то разделить его на 4, если не кратен, то умножить на два.
+
+list3 = [4, 3, 8, 10, 25]
+list_new = []
+
+for li in list3:
+    if li % 2 == 0:
+        new_item = li / 4
+        list_new.append(new_item)
```

Comment:
Смотрите, в программировании есть принцип DRY - Don't Repeat Yourself(не повторяйся). Это имеется ввиду что писать один и тот же код плохо. Тут делать append имеет смысл уже после того как вы получили new_itel, когда вышли из ввсех проверок по if, тогда бы строчка `list_new.append(new_item)` былв бы только один раз написана

Reply:
list3 = [4, 3, 8, 10, 25]
```suggestion
        list_new.append(new_item)
```
list_new = []

for li in list3:
    if li % 2 == 0:
        new_item = li / 4
    elif li % 2 != 0:
        new_item = li * 2
    list_new.append(new_item)

print("Новый список:", list_new)

Reply:
```suggestion
        list_new.append(new_item)
       
for li in list3:
    if li % 2 == 0:
        new_item = li / 4
    elif li % 2 != 0:
        new_item = li * 2
    list_new.append(new_item)
```